ckanext/dia_theme/templates/snippets/search_results_heading.html

```diff
@@ -0,0 +1,25 @@
+{#
```

Comment:
Can this search input etc be moved to the top of the whole page to match the layout of search on CWP please.